File: Java/01_06_2025/menentukanGanjilGenap.java
public class menentukanGanjilGenap {
    public static void main(String[] args) {
        java.util.Scanner input = new java.util.Scanner(System.in);

        System.out.print("Masukkan bilangan: ");
        int bilangan = input.nextInt();

        if (bilangan % 2 == 0) {
            System.out.println(bilangan + " adalah bilangan genap.");
        } else {
            System.out.println(bilangan + " adalah bilangan ganjil.");
        }

        input.close();
    }
}

File: Go/April & May/25-04-2025/menentukanGanjilGenap.go
package main

import "fmt"

func main1() {

	var input int;

	fmt.Print("Masukan sebuah angka: ")
	fmt.Scanln(&input)

	if input % 2 == 0 {
		fmt.Println(input, " adalah sebuah bilangan genap");
	} else{
		fmt.Println(input,  " adalah sebuah bilangan ganjil");
	}
}